doc/api/esm.md

```diff
@@ -722,6 +722,17 @@ const instance2 = await WebAssembly.instantiate(libraryModule, {
 });
 ```
 
+In addition to the static source phase, there is also a dynamic variant of the
+source phase via the `import.source` dynamic phase import syntax:
+
+```js
+const dynamicLibrary = await import.source('./library.wasm');
+
+const instance = await WebAssembly.instantiate(dynamicLibrary, {
+  custom: importObj,
```

Comment:
Is it clear what `importObj` is supposed to be? It's not to me, could we add a comment or an actual example?

Reply:
In WebAssembly instantiation, this is `instantiate` takes an import object as its second parameter corresponding to a record of key value pairs of specifier to module namespace objects. See https://developer.mozilla.org/en-US/docs/WebAssembly/Reference/JavaScript_interface/instantiate_static#syntax for more info. To align on the terminology I've simplified the parameter examples to just use the import object term directly instead of using an object literal.